src/com/gh4a/fragment/IssueListFragment.java

```diff
@@ -111,9 +110,7 @@ public void onItemClick(Issue issue) {
     @Override
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (requestCode == REQUEST_ISSUE) {
-            if (resultCode == Activity.RESULT_OK) {
-                super.onRefresh();
-            }
```

Comment:
Won't this also cause a refresh when canceling the 'new issue' activity with the back button?